wp-content/plugins/hurumap/src/Theme.js

```diff
@@ -50,7 +58,9 @@ export default createTheme({
           fontFamily: FONT_FAMILY_TEXT,
           fill: 'rgb(0,0,0)'
         }
-      }
+      },
+      fixLabelOverlap: true,
+      tickCount: 6
```

Comment:
6 would still a bit high for HURUmap data e.g. millions. We're using 3 in UI: https://github.com/TakwimuAfrica/Takwimu.AFRICA/blob/db745881ca9b32d387bce79d06e4a96e721589ea/src/theme.js#L61

Reply:
I have reduce this to 4